src/ray/raylet/main.cc

```diff
@@ -5,12 +5,13 @@
 
 #ifndef RAYLET_TEST
 int main(int argc, char *argv[]) {
-  RAY_CHECK(argc == 5);
+  RAY_CHECK(argc == 6);
 
   const std::string raylet_socket_name = std::string(argv[1]);
   const std::string store_socket_name = std::string(argv[2]);
   const std::string redis_address = std::string(argv[3]);
   int redis_port = std::stoi(argv[4]);
+  const std::string node_ip_address = std::string(argv[5]);
```

Comment:
Could we make this the first argument? Otherwise LGTM.

Reply:
Yes! Good idea :)